slk/repl/repl.py

```diff
@@ -1527,6 +1563,91 @@ def do_setup_accounts(self: SidechainRepl, line: str) -> None:
     # setup_accounts
     ##################
 
+    ##################
+    # load_batch
+
+    def do_load_batch(self: SidechainRepl, line: str) -> None:
+        """
+        Implementation of the `load_batch` REPL command.
+
+        Args:
+            line: The command-line arguments.
+        """
+        args = line.split()
+        if len(args) != 1:
+            print(
+                'Error: load_batch command takes exactly one argument. Type "help" '
+                "for help."
+            )
+            return
+        try:
+            path = Path(args[0])
+            if not path.is_file():
+                print(f'Error: no such file: {path}')
+                return
+            with open(path) as f:
+                for l in f:
+                    # remove comments. A comment is a '#' and anything that follows it up to the end of a line
+                    # Note this doesn't account for '#' inside strings or escaped, but that should be fine for this
+                    # simple repl. It doesn't have strings anyway.
+                    l = l.split('#')[0].strip()
+                    if not l:
+                        continue
+                    print(f'Running: {l}')
```

Comment:
nit:
```suggestion
                    print(f'Running: `{l}`')
```

Reply:
Fixed in `491c3e8 fixup! feat: Add load_batch command (#89)`